lib/api-wmata.js

```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+
+var props = require('../resources/properties');
+var secrets = require('../secret/secrets');
+
+var wmataReq = request.defaults({
+  baseUrl: props.wmata_base_url,
+  qs: secrets.apiKey
+});
+
+var WmataApi = function() {};
+
+WmataApi.prototype.get = function(endpoint, response, callback) {
+  wmataReq(endpoint, function(error, res, body) {
+    if (!error && res.statusCode === 200) {
+      callback(JSON.parse(body));
+    } else {
+      console.error(endpoint.concat(': Error with WMATA'), error);
+      response.tell(props.wmataErrorSpeechOutput);
+    }
+  });
+};
+
+module.exports = new WmataApi();
```

Comment:
What I was suggesting if we didn't make it an object is similar to `utils.js` module. We would have

    exports.get = function...
    // possibly in the future
    exports.getServiceInfo = function...
    
Then you use it like this

    var wmataApi = require('api-wmata.js');
    wmataApi.get('foo');

We're on the same page with the WmataApi object. I just realized there are wmata-api libraries, [this one](https://www.npmjs.com/package/node-wmata-metro) seems to be the most widely used.